packages/office-addin-cli/src/read-package-json-fast.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "read-package-json-fast" {
+    declare type ScriptsObject =  { [key: string]: string };
+    export default function readPackageJson(filePath: string): Promise<{ scripts: ScriptsObject, [key: string]: unknown}>;
```

Comment:
This provides better type info. The parsed package.json will return an object containing a scripts property, which is a an object containing keys and values that are strings. The object can contain other string properties with any type of property.

Reply:
I have never created a .d.ts file, I thought they were just auto generated, I will search and learn more about what this file does.

Reply:
JavaScript has no type info by default. TypeScript adds type info which helps catch errors during compilation rather than at runtime. The d.ts files can be generated by the TypeScript compiler, because after compilation the .js files that are output have no type info, and the type info from the original .ts files can be output to .d.ts. 

Since read-package-json-fast does not have type info, I've created read-package-json-fast to provide that type info.
https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html